### ISSUE TITLE
watch feature not watching for changes

### ISSUE THREAD

**Issue by bacongobbler** [Azure/draft#341]
It looks like the watch feature in Draft does not work at all. When I make changes to an application with watch enabled, no builds are reported or uploaded to tiller.

**Comment by pikeas** [comment]
Just discovered draft and this seems like a deal breaker for use in any real dev environment...

**Comment by squillace** [comment]
@pikeas yep, and we're just about to fix that, as I understand. Us, too. All though it turns out we get very different feedback from different dev types.

**Comment by bacongobbler** [comment]
removing this from the milestone in favour of the VSCode "draft up on save" experience. The existing file watcher libraries are all *very* buggy. VSCode has got the right experience.

See https://github.com/Azure/vscode-kubernetes-tools/pull/160 for a work-in-progress implementation running `draft up` when a file is saved.

**Comment by bacongobbler** [comment]
I'm almost in favour of removing the file watcher from Draft entirely and handing this off to the IDE to handle for us. That would be a good move forward for users: they get the experience they want, and we can revisit this when there are most stable file watcher libs available for use (or we write one ourselves).
